sdk/src/driftClient.ts

```diff
@@ -160,6 +160,7 @@ type RemainingAccountParams = {
 	readablePerpMarketIndex?: number | number[];
 	readableSpotMarketIndexes?: number[];
 	useMarketLastSlotCache?: boolean;
+	userAccountNotAvailable?: boolean;
```

Comment:
i think instead of adding this, you can just set useMarketLastSlotCache to false